yoti-sdk-spring-boot-auto-config/spotbugs/exclude-filter.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<FindBugsFilter>
+
+  <Match>
+    <Bug pattern="CRLF_INJECTION_LOGS"/>
```

Comment:
when excluding something, it's a good practice to explain why with a comment - is it a bug? a false positive? a rule that can be safely ignored in the particular project? e.g.
```xml
<!-- These patterns became completely broken after 4.3 - ignored them -->
<!-- See https://github.com/spotbugs/spotbugs/issues/1797 -->
<Bug pattern="EI_EXPOSE_REP,EI_EXPOSE_REP2"/>
```

Reply:
Sure. I didn't add any new exceptions tho, just renaming the legacy XMLs. Now that the build doesn't fail with the spotbugs violations I was almost about to drop all the exceptions to start from scratch. I'll see if I can add few.

Reply:
ah ok the diff looked like it was new nvm mind me then :+1: